content/reference/admin/private_locations/private_packages/index.md

```diff
@@ -9,71 +9,134 @@ weight: 22033
 
 # Introduction
 
-Using Private Locations allows you to run your simulations in your private cloud. Now, what about your simulations' packages? They may hold private information, so you may want to also store them in your private cloud.
+Private Locations facilitate running simulations in your dedicated cloud environment. 
+Ensure secure storage of sensitive simulation packages in your private cloud. 
 
-Your Control Plane has a private repository feature, allowing you to manage your simulation packages privately, you just have to enable it.
+The Control Plane offers a private repository; enable it for confidential package management!
 
 # Private packages
 
-## Infrastructure
+To create private packages, utilize our Gatling Enterprise Cloud packages API and upload them through a 
```

Comment:
I find `utilize` strange here, `use` may be more suitable. IIRC my English lessons (ouch, that was a while back), "utilize" may be translated as "tirer parti"